### ISSUE TITLE
welp, it happened again...

### ISSUE THREAD

**Issue by DuyLvzy123** [UnamSanctam/SilentCryptoMiner#803]
<img width="674" alt="image" src="https://user-images.githubusercontent.com/99333895/170931292-9a5096c0-c67e-46a4-b644-8297a775d066.png">

**Comment by UnamSanctam** [comment]
Are you completely sure that that's the miner you installed? Meaning that "haha.exe" is the miner you have installed?

**Comment by DuyLvzy123** [comment]
> Are you completely sure that that's the miner you installed? Meaning that "haha.exe" is the miner you have installed?
yes, 'm sure
Probably because my computer is bad

**Comment by UnamSanctam** [comment]
Only way it would become like that is if you either ran the uninstaller or disabled "Run miner after install" or if you downloaded some antivirus that removed it.

**Comment by DuyLvzy123** [comment]
> Only way it would become like that is if you either ran the uninstaller or disabled "Run miner after install" or if you downloaded some antivirus that removed it.

Oh I have avast antivirus bruhhh

**Comment by DuyLvzy123** [comment]
> Only way it would become like that is if you either ran the uninstaller or disabled "Run miner after install" or if you downloaded some antivirus that removed it.

So if I turn off run miner after install, will it have any effect?

**Comment by UnamSanctam** [comment]
That will just stop it from running after installation, I mentioned that option since I've had people open issues that it doesn't run but they had disabled "Run miner after install" so there wasn't actually any problem.

**Comment by DuyLvzy123** [comment]
> That will just stop it from running after installation, I mentioned that option since I've had people open issues that it doesn't run but they had disabled "Run miner after install" so there wasn't actually any problem.

I'll try using debug to see what's wrong with it

**Comment by DuyLvzy123** [comment]
> That will just stop it from running after installation, I mentioned that option since I've had people open issues that it doesn't run but they had disabled "Run miner after install" so there wasn't actually any problem.

<img width="527" alt="image" src="https://user-images.githubusercontent.com/99333895/170943186-b8b35851-3143-482b-8043-dc86c176470d.png">
bruh

**Comment by UnamSanctam** [comment]
That's just a cast warning, it doesn't affect anything

**Comment by UnamSanctam** [comment]
But if you have Avast then most likely it's just removing the miner.

**Comment by DuyLvzy123** [comment]
> But if you have Avast then most likely it's just removing the miner.

oh I forgot turn off real-time....

**Comment by DuyLvzy123** [comment]
> But if you have Avast then most likely it's just removing the miner.

shell code loader automatically turns off, due to new update?

**Comment by DuyLvzy123** [comment]
> Yes, currently I believe that having it disabled has the least amount of base detections (not depending on any crypts or obfuscations).

I think the victims will think how to delete that virus if found, the miner virus will no longer mine if the victim restarts the computer?

**Comment by UnamSanctam** [comment]
The watchdog will reinstall the miner if they remove it (as long as the watchdog is still running).

**Comment by DuyLvzy123** [comment]
> The watchdog will reinstall the miner if they remove it (as long as the watchdog is still running).

yess still working
<img width="707" alt="image" src="https://user-images.githubusercontent.com/99333895/171014169-4d73e40e-4f1c-43e1-b19c-0333ee27ce6e.png">
Will the victim's computer be easily damaged?

**Comment by UnamSanctam** [comment]
No, well mining itself doesn't really do any damage. Any damage that appears in big mining rigs (that you might've heard about it the news before) is only due to the high temperatures of those rigs (they have multiple ones running at 100% 24/7) or fans degrading due to much use.

**Comment by UnamSanctam** [comment]
I mean the computer has to be on if that's what you're wondering, but all that field is saying is that it cannot get the "Active Window" since the miner is running as the System user (and the System user has no active windows).

**Comment by DuyLvzy123** [comment]
> I mean the computer has to be on if that's what you're wondering, but all that field is saying is that it cannot get the "Active Window" since the miner is running as the System user (and the System user has no active windows).

<img width="245" alt="image" src="https://user-images.githubusercontent.com/99333895/171016662-dbcdfa98-c662-4f2c-bee1-7ca732d7bcfb.png">
I'm curious about this but I accidentally typed it in and added it, how do I remove it?

**Comment by UnamSanctam** [comment]
You can find it in the "Edit Configuration", then select it and press "Remove".

**Comment by DuyLvzy123** [comment]
> You can find it in the "Edit Configuration", then select it and press "Remove".

oh nevermind

**Comment by UnamSanctam** [comment]
Select the configuration you made and delete it, you can't delete the default configurations anyway so there's nothing to worry about.

**Comment by DuyLvzy123** [comment]
> Select the configuration you made and delete it, you can't delete the default configurations anyway so there's nothing to worry about.

ok

**Comment by DuyLvzy123** [comment]
> Select the configuration you made and delete it, you can't delete the default configurations anyway so there's nothing to worry about.

when will the webpanel update?
I bothered you, good night or good morning.

**Comment by UnamSanctam** [comment]
Today probably, but the current version also works fine in the meantime.

**Comment by DuyLvzy123** [comment]
> > Select the configuration you made and delete it, you can't delete the default configurations anyway so there's nothing to worry about.
> 
> when will the webpanel update? I bothered you, good night|good morning.



> Today probably, but the current version also works fine in the meantime.

I just asked but everything is fine, byeee
